$OpenBSD: patch-clutter_clutter_pc_in,v 1.1 2012/07/07 12:49:17 ajacoutot Exp $

XXX our pkg-config(1) cannot work with variables in Requires.

--- clutter/clutter.pc.in.orig	Sun Jun 17 09:42:37 2012
+++ clutter/clutter.pc.in	Sun Jun 17 09:43:00 2012
@@ -4,8 +4,6 @@ libdir=@libdir@
 includedir=@includedir@
 
 apiversion=@CLUTTER_API_VERSION@
-requires=@CLUTTER_REQUIRES@
-requires_private=@CLUTTER_REQUIRES_PRIVATE@
 backends=@CLUTTER_BACKENDS@
 
 # only kept for backward compatibility
@@ -20,5 +18,5 @@ Description: Clutter Core Library
 Version: @VERSION@
 Libs: -L${libdir} -lclutter-${apiversion}
 Cflags: -I${includedir}/clutter-${apiversion}
-Requires: ${requires}
-Requires.private: ${requires_private}
+Requires: @CLUTTER_REQUIRES@
+Requires.private: @CLUTTER_REQUIRES_PRIVATE@
